RetrospectiveExtension.Frontend/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Azure DevOps Retrospective Extension",
   "main": "index.tsx",
   "scripts": {
-    "build:d": "rm -rf ./dist && webpack --mode=development",
-    "build:p": "rm -rf ./dist && webpack --mode=production",
+    "build:d": "rm -rf ./dist && webpack --mode=development ",
```

Comment:
nit: can you remove these redundant spaces on this line and the one below?

Reply:
done